src/scripts/migrate.py

```diff
@@ -124,7 +135,7 @@ def slurp(
             terms_to_slurp.append({
                 'mondo_id': mondo_id, 'mondo_label': mondo_label, 'xref': t.curie, 'xref_source': 'MONDO:equivalentTo',
                 'original_label': t.label if t.label else '', 'definition': t.definition if t.definition else '',
-                'parents': '|'.join(qualified_parent_mondo_ids)})
+                'parents': '|'.join([p for p in t.direct_owned_parent_curies])})
```

Comment:
@matentzn We are now removing the need for checking anything regarding a term's parents to determine migration candidacy. As such I needed to update the `parents` column in `slurp/%.tsv`. Previously, this was only for `qualified_parent_mondo_ids`. Now it will show all parents.

Reply:
I don't understand this at all! Sorry :/ its better to communicate with me in output tables then in code..

The parent column in the migration table (the one processed by Sabrina) can only contain exact or narrow-matched parents, so I am not sure if this is still true after this change.

Reply:
Ash, OK. Your comment [here](https://github.com/monarch-initiative/mondo-ingest/pull/316#discussion_r1199793432) clarifies what you mean. Will continue discussion there.